en/developer_guide/external_media_domain.rst

```diff
@@ -0,0 +1,25 @@
+Adding an external media domain
+===============================
+
+Configuration
+~~~~~~~~~~~~~
+
+Sometimes, developers want an external media domain to improve websites performances.
+Open Orchestra gives you an easy way to add an external media domain in the configuration:
+
+.. code-block:: yaml
+
+    open_orchestra_media:
+        media_domain: http://exemple.com
```

Comment:
You should use a more accurate example as http://my-media-domain.com or something like this.